src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    private final int maxValue = 100;

```

Comment:
```suggestion
    private static final int MAX_VALUE = 100;
```
it should be constant - static + constant case is lacking